src/taipy/core/config/data_node_config.py

```diff
@@ -1030,6 +1052,59 @@ def _configure_mongo_collection(
             id, DataNodeConfig._STORAGE_TYPE_VALUE_MONGO_COLLECTION, scope, validity_period, **properties
         )
 
+    @classmethod
+    def _configure_s3_object(
+        cls,
+        id: str,
+        aws_access_key: str,
+        aws_secret_acces_key: str,
+        aws_s3_bucket_name: str,
```

Comment:
```suggestion
        aws_s3_bucket_name: str,
        aws_s3_object_key: str,
```

The `aws_s3_object_key` is missing right?